test/controllers/v1/rules_controller_test.rb

```diff
@@ -41,9 +41,12 @@ class RulesControllerTest < ActionDispatch::IntegrationTest
       assert_response :success
     end
 
-    test 'finds rules not within the user scope but within latest' do
+    test 'finds latest canonical rules' do
+      assert(profiles(:one).canonical?)
+      profiles(:one).rules << rules(:two)
+      assert(rules(:two).canonical?)
       assert_includes(Rule.latest, rules(:two))
-      assert_not_includes(::Pundit.policy_scope(users(:test), ::Rule),
+      assert_not_includes(accounts(:test).profiles.map(&:rules).uniq,
```

Comment:
Here's the failing rule test due to adding the pundit policy for finding canonical rules by slug.